### ISSUE TITLE
Unable to reproduce results

### ISSUE THREAD

**Issue by robmarkcole** [satellogic/iquaflow-mnist-use-case#2]
Hi
I noticed the parameters in `IQF-UseCase.ipynb` will not produce the final plot. I have tried to reproduce but my plot results look quite different, with larger variation and noise_0 having highest loss. Can you advise how to reproduce the results?

```python3
ds_modifiers_list = [
    CustomNoiseModifier(params = {"sigma": 1*f})
    for f in [0,0.5,1,2,4,8, 16, 32] # add 16, 32
]

experiment = ExperimentSetup(
    experiment_name   = experiment_name,
    task_instance     = task,
    ref_dsw_train     = ds_wrapper_train,
    ref_dsw_val       = ds_wrapper_valid,
    ds_modifiers_list = ds_modifiers_list,
    repetitions       = 5 # increase from 1
)
```

<img width="576" alt="image" src="https://user-images.githubusercontent.com/11855322/196378833-e17bc502-c87d-4630-89d2-9c73d9078eb8.png">

**Comment by paugalles** [comment]
Yes, you are correct. The plot was not updated according to the latest setup.
The only randomness is fixed in this line [here](https://github.com/satellogic/iquaflow-mnist-use-case/blob/main/custom_train.py#L31). Thus the solution is deterministic.

Since I am updating the use case as per #3 I will update the plot there.
